rollup.config.mjs

```diff
@@ -98,26 +98,42 @@ export default [
 			}
 		]
 	},
-	/* compiler.js */
+	// compiler.js - ESM version
```

Comment:
It would be great if we didn't need the ESM version at all - just unbundled. Is there something stopping is from doing that?
If there is then that begs the question if it's worthwhile to even do the TS->JSdoc conversion.

Reply:
you mean just run `tsc` to convert the `.ts` files to `.js` without having rollup involved for the ESM version? it might be possible. if you didn't do the TS->JSdoc conversion it might make things heavier if you distributed sourcemaps. you'd also have to do a build step before running the tests. but anyway, all that seems like a larger conversation separate from this change

Reply:
PR for unbundling here: https://github.com/sveltejs/svelte/pull/8613